etc/skel/.config/hypr/hyprland.conf

```diff
@@ -68,289 +94,369 @@ decoration {
+    workspace_back_and_forth = 1
+    workspace_center_on = 1
+	movefocus_cycles_fullscreen = true
+	window_direction_monitor_fallback = true
 }
 
-device {
-    name = epic-mouse-v1
-    sensitivity = -0.5
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#xwayland
+xwayland {
+	enabled = true
+	use_nearest_neighbor = true
+	force_zero_scaling = false
 }
 
-# See https://wiki.hyprland.org/Configuring/Keywords/ for more
-$mainMod = SUPER
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#opengl
+opengl {
+	nvidia_anti_flicker = true
+	force_introspection = 2
+}
 
-# Example binds, see https://wiki.hyprland.org/Configuring/Binds/ for more
-bind = $mainMod, RETURN, exec, alacritty
-bind = $mainMod, A, exec, grim -g "$(slurp)" - | swappy -f -
-bind = $mainMod, Q, killactive,
-bind = $mainMod SHIFT, M, exit,
-bind = $mainMod, V, togglefloating,
-bind = CTRL, SPACE, exec, rofi -show combi -modi window,run,emoji,combi -combi-modi window,run,emoji
-#bind = CTRL, SPACE, exec, wofi
-bind = $mainMod, F, fullscreen
-bind = $mainMod, Y, pin
-#bind = $mainMod, P, pseudo, # dwindle
-bind = $mainMod, J, togglesplit, # dwindle
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#render
+render {
+   explicit_sync = 2
+   explicit_sync_kms = 2
+   direct_scanout = true
+   expand_undersized_textures = true
+}
 
-bind = $mainMod, K, togglegroup,
-bind = $mainMod, Tab, changegroupactive, f
+# https://wiki.hyprland.org/0.45.0/Configuring/Variables/#cursor
+cursor {
+	no_hardware_cursors = 2
+	allow_dumb_copy = true
+	enable_hyprcursor = true
+	hide_on_touch = true
+	# default_monitor = $exter
+	sync_gsettings_theme = true
+}
+# End of Variables Section
 
-bind = $mainMod SHIFT, G,exec,hyprctl --batch "keyword general:gaps_out 5;keyword general:gaps_in 3"
-bind = $mainMod , G,exec,hyprctl --batch "keyword general:gaps_out 0;keyword general:gaps_in 0"
+# See https://wiki.hyprland.org/0.45.0/Configuring/Dwindle-Layout/ for more
+dwindle {
+    force_split = 0
+    special_scale_factor = 0.8
+    split_width_multiplier = 1.0
+    use_active_for_splits = true
+    pseudotile = yes # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
+    preserve_split = yes
+}
 
-# Volume control
+# See https://wiki.hyprland.org/0.45.0/Configuring/Master-Layout/ for more
+master {
+    new_status = master
+    special_scale_factor = 0.8
+    inherit_fullscreen = true
+    smart_resizing = true
+    drop_at_cursor = true
+}
 
-bind=,XF86AudioLowerVolume,exec,pamixer -ud 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
-bind=,XF86AudioRaiseVolume,exec,pamixer -ui 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
-# mute sound
-bind=,XF86AudioMute,exec,amixer sset Master toggle | sed -En '/\[on\]/ s/.*\[([0-9]+)%\].*/\1/ p; /\[off\]/ s/.*/0/p' | head -1 > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
+# https://wiki.hyprland.org/0.45.0/Configuring/Keywords/#per-device-input-configs
+device {
```

Comment:
Remove the entire device part or leave it commented because it should be up to the user to make his adjustements

Reply:
Those are the defaults from /usr/share/hypr/hyprland.conf

Reply:
All fine then.